templates/patterns/page_header/pattern-page-header.html.twig

```diff
@@ -4,11 +4,47 @@
  * Default implementation for the page header component.
  */
 #}
-{% include '@ecl/ec-component-page-header' with {
-  'identity': identity,
-  'breadcrumb': breadcrumb,
-  'language_switcher': language_switcher,
+
+{% set _meta = '' %}
+{% for meta in metas %}
+  {% if _meta != '' %}
+    {% set _meta = _meta ~ ' | ' %}
+  {% endif %}
+  {% set _meta = _meta ~ meta %}
+{% endfor %}
+
+{% set _breadcrumb_links = {} %}
+{% for breadcrumb_item in breadcrumb %}
+  {% set _links = {
+    'label' : breadcrumb_item.label,
+    'path' : breadcrumb_item.href
+  } %}
+  {% set _breadcrumb_links = _breadcrumb_links|merge([_links]) %}
+{% endfor %}
+{% set _breadcrumb = {
+  'links' : _breadcrumb_links,
+  'navigation_text' : '',
+  'ellipsis_label': '',
+  'icon_file_path' : ecl_icon_path
+} %}
+
+{% set _infos = {} %}
```

Comment:
See comment above, we should derive the info below from the info type.

Reply:
We don't need to change the name of this variable.
see https://ecl-twig.netlify.com/ec/?path=/story/components-page-header--title
```
breadcrumb: {  
    links: [    
      {  
        label: 'Link 1',  
        path: '/example'  
      },  
    ],  
    icon_file_path: '/path-to-the-icons-file',  
    navigation_text: 'You are here:',  }, 
```

Reply:
Here, it's not an icon component, only a param of the breadcrumb.

Reply:
Sure, sorry I meant something else! :D Just put the comment on the wrong line.